engine/engine_manager.go

```diff
@@ -255,6 +255,31 @@ func (em *Manager) EngineUpgrade(ctx context.Context, req *rpc.EngineUpgradeRequ
 	return el.RPCResponse(response.Status), nil
 }
 
+func (em *Manager) EngineLog(req *rpc.LogRequest, srv rpc.LonghornEngineManagerService_EngineLogServer) error {
+	logrus.Infof("Engine Manager getting logs for engine %v", req.Name)
+
+	em.lock.RLock()
+	defer em.lock.RUnlock()
+
+	el, exists := em.engineLaunchers[req.Name]
+	if !exists {
+		return fmt.Errorf("cannot find engine %v", req.Name)
+	}
+
+	el.lock.RLock()
+	err := el.engineLog(&rpc.LogRequest{
```

Comment:
Is it possible to get all the logs happened in this engine launcher instead of just the current one?

Reply:
I am going to merge this now. Let's deal with launcher's engine logs later.